tests/testthat/test-visr_plot.R

```diff
@@ -109,7 +108,7 @@ testthat::test_that("T1.3 An error when applied to an object that is not `survfi
   
 })
 
-# Requirement T2 ---------------------------------------------------------------
+# Requirement T2 -------------------------------------------------------------------------------------------------------
```

Comment:
Why the longer ---------------?
The section inserted by Rstudio itself are of a specific length and I would prefer to keep them as they are for consistency

Reply:
Oh okay, didn't know they were RStudio formatting. I just extended them to fill the 120 char limit. RStudio seems to use 80 chars

Reply:
Good point with the pdf render output